Implemented 2 new hud_itemsclock styles:

tcsabina opened the pull request:
Style 4: progress bar, where item name and counter are in the bar
Style 5: progress bar, where only the item time is in the bar

meag commented:
Hi Toma

Can you make these styles 6 & 7 please?  4 & 5 already exist in 3.2 (https://github.com/ezQuake/ezquake-source/blob/master/mvd_utils.c#L784-L818), 3.5 is still based on 3.1 at the moment (will try and get this fixed soon).

Also - it looks like maybe you used tabs for alignment at the top of the file, can you change those to spaces please?

Cheers,
meag

tcsabina commented:
Hi meag,

I've replaced the tabs with spaces in the mvd_wp_info[] array, and also modified the style numbers.
The reason I used tabs in the first place is that it is used everywhere in the file, instead of spaces. But no probs, from now on it will be spaces...

I could replace all tabs with 4 spaces in this file, but that would create too much noise...


regards,
toma
